website/docs/documentation/Pipelines/Optimus_Pipeline/Loom_schema.md

```diff
@@ -98,3 +98,32 @@ The global attributes in the Loom apply to the whole file, not any specific part
 |`molecules_with_single_read_evidence`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of molecules associated with this entity that are observed by only one read. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.CellMetrics.molecules_with_single_read_evidence)|
 |`number_cells_detected_multiple`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of cells which observe more than one read of this gene. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.GeneMetrics.number_cells_detected_multiple)|
 |`number_cells_expressing`|[SC Tools](https://github.com/HumanCellAtlas/sctools/tree/master/src/sctools/metrics)| The number of cells that detect this gene. [Metrics Definitions](https://sctools.readthedocs.io/en/latest/sctools.metrics.html#sctools.metrics.aggregator.GeneMetrics.number_cells_expressing)|
+
+## HCA Data Coordination Platform Matrix Processing
+Optimus was designed for processing data from the Human Cell Atlas (HCA) Data Coordination Platform (DCP; see the DCP [Data Portal here](https://data.humancellatlas.org/)). 
```

Comment:
```suggestion
Optimus supports data processing for the Human Cell Atlas (HCA) Data Coordination Platform (DCP; see the DCP [Data Portal here](https://data.humancellatlas.org/)). 
```

It was originally developed for HCA but I don't want to imply that's all it's good for.